### ISSUE TITLE
Questions about experimental results

### ISSUE THREAD

**Issue by zw615** [tiancity-NJU/da-faster-rcnn-PyTorch#12]
Hi, I have some questions regarding the performance of your code.
The original paper's performance ( using the  Cityscapes training set  as the source domain and the  Foggy Cityscapes validation set as the target domain) is  lower than your code.

original:
![图片](https://user-images.githubusercontent.com/26880977/58847384-5680e080-86b5-11e9-9be5-625763fb300c.png)

yours:
 final mAP=30.71% on baseline mAP=24.26%

I wonder why. Did you add some additional tricks or the implementation of original paper is buggy? Also, can your result be reproduced stably?
Wonderful work, by th way. Thanks a lot

**Comment by codingwolfman** [comment]
@JWarlock i think 30.71% is result of cityscape to foggy cityscape，still，the result is higher than paper

**Comment by mumu999701** [comment]
Could someone reproduce the code of the warehouse, I got the mean ap only 0.1061
